entity-framework/core/querying/raw-sql.md

```diff
@@ -43,29 +46,29 @@ The following example passes a single parameter to a stored procedure. While thi
 var user = "johndoe";
 
 var blogs = context.Blogs
-    .FromSql("EXECUTE dbo.GetMostPopularBlogsForUser {0}", user)
+    .FromSqlInterpolated("EXECUTE dbo.GetMostPopularBlogsForUser {0}", user)
     .ToList();
 ```
 
-This is the same query but using string interpolation syntax, which is supported in EF Core 2.0 and above:
+This is the same query but using named interpolation syntax:
 
 <!-- [!code-csharp[Main](samples/core/Querying/Querying/RawSQL/Sample.cs)] -->
 ``` csharp
 var user = "johndoe";
 
 var blogs = context.Blogs
-    .FromSql($"EXECUTE dbo.GetMostPopularBlogsForUser {user}")
+    .FromSqlInterpolated($"EXECUTE dbo.GetMostPopularBlogsForUser {user}")
     .ToList();
 ```
 
-You can also construct a DbParameter and supply it as a parameter value:
+You can also construct a DbParameter and supply it as a parameter value. Since a regular SQL parameter placeholder is used, rather than a string placeholder, `FromSqlRaw` can be safely used:
```

Comment:
I don't get why this would be considered a reason it is safe to use `FromSqlRaw`.